pdb2sql/align.py

```diff
@@ -0,0 +1,95 @@
+import numpy as np
+from .pdb2sqlcore import pdb2sql
+from .transform import rot_xyz_around_axis
+
+
+def align(pdb, axis='z', export=True, **kwargs):
```

Comment:
It would be nice if we provide an `orient` function to align the first three principal axes to x, y and z axes.

Reply:
is it necessary ? that's a lot of code that I think we'll never use. Is it something that is regularly done in bioinformatics ?

Reply:
Yes, it's common, see examples below.  You are adding the align-to-plane function, and I guess  the plane is by default  defined as the positive vectors, e.g. [1,0,0] and [0,1,0], so actually it will have the same effect as `orient`. 
 
https://pymolwiki.org/index.php/Orient,
https://nmr.cit.nih.gov/xplor-nih/xplorMan/node159.html, https://salilab.org/modeller/9v4/manual/node180.html

Reply:
I've opened an issue for that #28. I don't think it is critical atm and we can revive that when we have a bit more time